pkgs/servers/fiche/default.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  pname = "fiche";
+  version = "0.9.1";
+
+  src = fetchurl {
+    url = "https://github.com/solusipse/${pname}/archive/${version}.tar.gz";
+    sha256 = "TODO";
```

Comment:
If you try to built this locally you will get the Sha that needs to be put here.

Reply:
Done.